make.js

```diff
@@ -439,6 +587,53 @@ function getInternalPropertyType(property, prefix) {
     return type;
 }
 
+function getPublicPropertyTypeClassName(property, prefix) {

```

Comment:
>getPublicPropertyTypeClassName [](http://example.com/codeflow?start=9&length=30)

how is this different from the "getPublicPropertyType" function below?

Reply:
one includes const * and other wacky stuff that can't be found in class map, so needed another function

Reply:
could you take an additional parameter and make it one function? At the very least clarify the difference in a comment.  Looks like a lot of duplication and its not clear when one should be used over the other.

Reply:
ya, i can merge them

---
In reply to: [635604914](https://github.com/jplafonta/XPlatCTemplate/pull/19#discussion_r635604914) [](ancestors = 635604914)

Reply:
do other usages of this function need to be updated to pass the new param? or will it just work?